### PR TITLE
Moving email acknowledgement button and adding functionality

### PR DESCRIPTION
**Pull request by labib08**
Moved the "Email Acknowledgement" button from Pre-Internship Student evaluation to New Intake WIP and added functionality that opens email in order to send the acknowledgement email to the selected students.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -36,8 +36,15 @@
...
+I will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.%0D%0A%0D%0A
+You can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.%0D%0A%0D%0AEach project page has a rough description of the role outline, but the key information is the 'Further Reading' section that may be available for that project.%0D%0A%0D%0A
+Please also be acquainted with the program handbook.%0D%0A%0D%0ADetails of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.%0D%0A%0D%0ADetails of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.%0D%0A%0D%0AA FAQ is also available at https://wehi-researchcomputing.github.io/faq">Copy Email Acknowledgement</a>
+        -->
+        <a href="#" id="emailAcknowledgementBtn">Email Acknowledgement</a>
+        {% endif %}
 {% endblock %}
```
Comment (Xenotion) [inline templates/index.html:48]: probably good idea to move the anchor tag button within /new_intake path as the button will only show up when a user enters list of new intake students table

File: templates/index.html [hunk 2]
```diff
@@ -332,8 +339,51 @@
...
+                    var mailtoLink = "mailto:?subject=Acknowledgement of your application to the WEHI student internship program&body=Hi All,%0D%0A%0D%0AThank you for applying for the WEHI Student Internship program.%0D%0A%0D%0AI will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.%0D%0A%0D%0AYou can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.%0D%0A%0D%0AEach project page has a rough description of the role outline, but the key information is the 'Further Reading' section that may be available for that project.%0D%0A%0D%0APlease also be acquainted with the program handbook.%0D%0A%0D%0ADetails of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.%0D%0A%0D%0ADetails of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.%0D%0A%0D%0AA FAQ is also available at https://wehi-researchcomputing.github.io/faq" + "&to=" + emailString;
```
Comment (Xenotion) [inline templates/index.html:359]: can possibly take this string and assign it to a separate variable and store it there to reduce congestion within the function

File: templates/index.html [hunk 3]
```diff
@@ -332,8 +339,51 @@
...
+
+    Please also be acquainted with the program handbook.
+
+    Details of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.
+
+    Details of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.
+
+    A FAQ is also available at https://wehi-researchcomputing.github.io/faq
+
+    </div>
```
Comment (Xenotion) [inline templates/index.html:387]: This might not be necessary anymore. I think it only existed to be used a template for copying to clipboard

**Comment by Xenotion** [comment]
Apart from minor fixes, looks good to me

**Review by Xenotion — CHANGES_REQUESTED** [review]
Some changes needed

**Comment by rowlandm** [comment]
This looks really good! Could we use the update email and into this PR before I approve it though? 

Hi All, 

Thank you for applying for the WEHI Student Internship program. 

I will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.  

If I am calling and it is a bad time, don’t answer. I can leave a message and call back later. 

To make things clearer, I have provided the questions I will be asking in the call. Please look through the questions and the website to be prepared for the call. 

1. How do I pronounce your name? I may ask you this question to ensure I do the best I can to pronounce your name. 
2. This is a 100% remote internship. This means all meetings are virtual. Are you OK with this? 
3. We take Diversity, Equity, and Inclusion seriously. We have a Code of Conduct that you have to abide by that is in the WEHI RCP Student Internship Handbook on page 10. Have you read it and are you willing to abide by it? 
4. Are you a talker or are you more of a listener? The reason I ask is because I don't want a team of all listeners I need at least one talker. 
5. Are you more of a facilitator/organiser or a follower? The reason I ask is because I don't want a team full of followers.  
6. And are you more of a person who's an intentional thinker where you will need to go away think about it or are you more of a quick brainstormer who likes to share ideas in a meeting? 
7. What projects are you interested in and why? 
8. I will then show you the projects that are currently still available based on your interests. 
9. I will then explain the next steps to you 
10. I will ask you if you have any other questions. 

Please be acquainted with the program handbook at
https://doi.org/10.6084/m9.figshare.21259467. 

Please have a look through all the projects at https://wehi-researchcomputing.github.io/project-wikis  

Other key documents are available at https://wehi-researchcomputing.github.io/students#key-documents-to-review-and-faq 

More details on how to apply are here: https://wehi-researchcomputing.github.io/how-to-apply 

A FAQ is also available at https://wehi-researchcomputing.github.io/faq

**Pull request merged**